cmd/traffic/cmd/manager/managerutil/systema.go

```diff
@@ -15,6 +16,10 @@ type SystemaCRUDClient interface {
 	a8rcloud.Closeable
 }
 
+func AgentIsProprietary(ctx context.Context) bool {
```

Comment:
The term "proprietary" is about ownership. Could this method be named "AgentIsLicensed"?